README.md

```diff
@@ -88,6 +92,38 @@ $ export updates_repo_url="<updates_repo_url_to_package_updates>"
 $ make rhel_iso
 ```
 
+<a name="releasing-minishift-iso"></a>
+## Releasing Minishift ISO
+
+<a name="create-a-release-notes-pr"></a>
+### Create a Release Notes PR
```

Comment:
I don't think we need sub sections for each points. Can we not just use an enumeration? This blows the table of content out so much and each section does not contain much.